server/schedule/labeler/labeler.go

```diff
@@ -96,16 +96,12 @@ func (l *RegionLabeler) adjustRule(rule *LabelRule) error {
 
 	switch rule.RuleType {
 	case KeyRange:
-		r, err := l.adjustKeyRangeRule(rule.Rule)
-		if err != nil {
-			return err
-		}
-		rule.Rule = r
-		return nil
-	case KeyRanges:
-		rules, ok := rule.Rule.([]map[string]interface{})
+		rules, ok := rule.Rule.([]interface{})
```

Comment:
How about change it to `rule.Rule.([]interface{})`

Reply:
but it is not required to be plural. This field contains the rule logic of the rule. Its content type depends on the rule-type. Maybe we can use some abstract word, like `Data` or `Detail`.

Reply:
make sence

Reply:
Will it be changed in this PR?

Reply:
use `Data`?

Reply:
ok